js-commit-guidelines.md

```diff
@@ -0,0 +1,99 @@
+# Git Commit Guidelines
+
+We have very precise rules over how our git commit messages can be formatted.
+This leads to **more readable messages** that are easy to follow when
+looking through the **project history**.  But also,
+we use the git commit messages to **generate the change log**.
```

Comment:
I think it's important to keep the emphasis on theses, but for that we should use italics rather than bold. Would that be okay for you?